org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,211 @@
+{
+    "app-id": "org.freedesktop.fwupd",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "branch": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/libexec/fwupd/fwupdtool",
```

Comment:
Would a desktop file with Terminal: true and this exec make sense for the context of use? Or is this in fact something that you'd prefer people didn't use (e.g. are the instructions going to be:
> flatpak run --command=bash org.freedesktop.fwupd

Reply:
I think it doesn't provide any terminal UI to make it work like this.

Reply:
Yea desktop files are for TUIs not CLIs.

Reply:
You really want to run this on the console, not with an icon.